src/components/pages/AdminMeetingsDashBoard/CrudModals/CreateModal.js

```diff
@@ -0,0 +1,215 @@
+import React, { useEffect, useState } from 'react';
+import 'antd/dist/antd.css';
+import { Button, Modal } from 'antd';
+import useAxiosWithAuth0 from '../../../../hooks/useAxiosWithAuth0';
+
+import DynamicDropdown from '../DynamicDropdown';
+
+const CreateModal = props => {
+  const axiosWithAuth = useAxiosWithAuth0();
+  const { data, setMeetings, meetings } = props;
+  const [allMentors, allMentees] = data;
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  const [formData, setFormData] = useState({
+    meeting_topic: '',
+    meeting_start_time: '',
+    meeting_end_time: '',
+    mentor_id: '',
+    mentee_id: '',
+    admin_meeting_notes: '',
+    mentor_meeting_notes: '',
+    mentee_meeting_notes: '',
+  });
+
+  const mentorsArray = props.data[0];
+  const menteesArray = props.data[1];
+  const processedMentorsArray = mentorsArray.map(mentor => {
+    const entry = {
+      value: mentor.profile_id,
+      label: mentor.profile_id,
+      name: 'mentor_id',
+    };
+
+    return entry;
+  });
+
+  const processedMenteesArray = menteesArray.map(mentee => {
+    const entry = {
+      value: mentee.profile_id,
+      label: mentee.profile_id,
+      name: 'mentee_id',
+    };
+
+    return entry;
+  });
+
+  const handleChange = e => {
+    setFormData({ ...formData, [e.target.name]: e.target.value });
+  };
+  const handleOk = e => {
+    e.preventDefault();
+    const randomMeetingId = Math.floor(Math.random() * 100000000000000000);
+    const meeting = {
+      meeting_id: randomMeetingId,
+      meeting_topic: formData.meeting_topic,
+      meeting_start_time: formData.meeting_start_time,
+      meeting_end_time: formData.meeting_end_time,
+      mentor_id: formData.mentor_id,
+      mentee_id: formData.mentee_id,
+      admin_meeting_notes: formData.admin_meeting_notes,
+      mentor_meeting_notes: formData.mentor_meeting_notes,
+      mentee_meeting_notes: formData.mentee_meeting_notes,
+    };
+    axiosWithAuth
+      .post(`${process.env.REACT_APP_API_URI}meetings/`, meeting)
+      .then(setMeetings([...meetings, meeting]))
+      .catch(err => console.error(err));
+
+    setIsModalOpen(false);
+  };
+
+  const showModal = () => {
+    setIsModalOpen(true);
+  };
+
+  const handleCancel = () => {
+    setIsModalOpen(false);
+  };
```

Comment:
add empty line between 78 - 79 to follow format of the other functions in this file